install_files/ansible-base/usr.sbin.apache2

```diff
@@ -188,6 +196,8 @@
   /var/www/securedrop/worker.py r,
   /var/www/securedrop/worker.pyc rw,
   /var/www/source.wsgi r,
+  /var/www/.gnupg/ rw,
```

Comment:
I can reproduce reliably on submission to the source interface. I don't know why it generated since we use a non-default homedir but every time it is generated

```
Feb  7 00:37:04 app-staging kernel: [13060.528666] audit: type=1400 audit(1423269424.913:22): apparmor="DENIED" operation="mkdir" profile="/usr/sbin/apache2" name="/var/www/.gnupg/" pid=4286 comm="gpg-agent" requested_mask="c" denied_mask="c" fsuid=33 ouid=33
Feb  7 00:37:04 app-staging kernel: [13060.552536] audit: type=1400 audit(1423269424.937:23): apparmor="DENIED" operation="mkdir" profile="/usr/sbin/apache2" name="/var/www/.gnupg/" pid=4293 comm="gpg-agent" requested_mask="c" denied_mask="c" fsuid=33 ouid=33
```